### PR TITLE
feat(defaultAggregate): Apply default aggregate functions in query Bu…

### PR DESCRIPTION
**Pull request by ebb-tide**
This PR will need to follow: https://github.com/influxdata/influxdb/pull/19129

### CODE REVIEW

File: ui/cypress/e2e/queryBuilder.test.ts [hunk 1]
```diff
@@ -89,6 +89,45 @@ describe('The Query Builder', () => {
       cy.contains('Basic Ole Dashboard').should('exist')
     })
 
+    it('when it creates a query, the query has an aggregate window, clicking around aggregate window selections work', () => {
+      cy.get('@org').then((org: Organization) => {
+        cy.visit(`orgs/${org.id}/data-explorer`)
+      })
+
+      cy.contains('mem').click('right') // users sometimes click in random spots
```
Comment (drdelambre) [inline ui/cypress/e2e/queryBuilder.test.ts:97]: i didn't know this ish existed! super cool

File: ui/src/shared/components/DurationInput.tsx [hunk 2]
```diff
@@ -63,9 +63,13 @@ const DurationInput: FC<Props> = ({
...
 
-  const inputStatus = isValid(inputValue)
-    ? ComponentStatus.Default
-    : ComponentStatus.Error
+  let inputStatus = controlledStatus || ComponentStatus.Default
+
+  if (inputStatus === ComponentStatus.Default) {
+    inputStatus = isValid(inputValue)
+      ? ComponentStatus.Default
+      : ComponentStatus.Error
```
Comment (drdelambre) [inline ui/src/shared/components/DurationInput.tsx:71]: isn't this just:
```
if (inputStatus === ComponentStatus.Default && !isValid(inputValue)) {
    inputStatus = ComponentStatus.Error
}
```

Comment (ebb-tide) [reply ui/src/shared/components/DurationInput.tsx:71]: it.. is? thanks !

File: ui/src/timeMachine/actions/queryBuilder.ts [hunk 3]
```diff
@@ -413,28 +439,50 @@ export const selectTagValue = (index: number, value: string) => (
...
-  } = state
-  const {draftQueries, activeQueryIndex} = getActiveTimeMachine(state)
+  const {draftQueries, activeQueryIndex} = getActiveTimeMachine(getState())
 
   const functions = draftQueries[activeQueryIndex].builderConfig.functions
 
   let newFunctions: BuilderFunctionsType[]
 
   if (functions.find(f => f.name === name)) {
+    if (functions.length == 1) {
```
Comment (drdelambre) [inline ui/src/timeMachine/actions/queryBuilder.ts:453]: doesn't this always return true since the find operation is being run before it?

Comment (ebb-tide) [reply ui/src/timeMachine/actions/queryBuilder.ts:453]: it doesn't, but it's confusing so I will assign some well named variables  :)

**Pull request merged**